app/assets/stylesheets/charactershow.css.scss

```diff
@@ -0,0 +1,247 @@
+/*Character show page*/
+
+.character-info {
+  @include outer-container;
+  padding: 20px;
+  border-top: 1px solid grey;
+  margin-top: 20px;
+}
+
+.sub-stats {
+  @include span-columns(5);
+  @include shift(1);
+  display: table;
+}
+
+.sub-stats li {
+  display: table-row;
+}
+
+.label {
+  display: table-cell;
+  width: 200px;
+}
+
+#character-name {
+  text-align: center;
+  font-size: 1.6em;
+  padding-top: 10px;
+  width: 30%;
+  margin-left: 35%;
+  margin-right: 35%;
+}
+
+.char-nav {
+  text-align: center;
+  border-bottom: 1px solid grey;
+  padding-bottom: 10px;
+  margin-bottom: 18px;
+}
+
+.char-nav li  {
+  list-style: none;
+  display: inline-block;
+
+}
+
+.char-nav li  a {
+  text-decoration: none;
+  color: black;
+}
+
+#barscilwe {
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-top: 20px;
+}
+
+#barscilwe tr td{
+  padding-left: 1.5%;
+  padding-right: 1.5%;
+  padding-top: 5px;
+  display: inline;
+}
+
+#new-char {
+  text-align: center;
+  padding: 20px 0px;
+  min-width:175px;
+  margin-top: 5%;
+}
+
+#new-char a {
+  text-decoration: none;
+  color: black;
+  padding: 5px 15px;
+  border: 2px ridge #029870;
+  border-radius: 15px;
+  background-color: rgba(2,152,112,0.1);
+  margin-top: 100px;
+}
+
+.content {
+  margin: 50px;
+}
+
+.char-list {
+  @extend %default-ul;
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.char-list li a {
+  text-decoration: none;
+  color: black;
+}
+
+h2 {
+  text-align: center;
+  font-size: 2.8em;
+}
+
+.user-status {
+  text-align: right;
+  display: inline;
+}
+
+.sub-heading {
+  font-weight: bold;
+}
+
+.edit-container{
+  @include outer-container;
+}
+
+.editing-characters{
+  @include editing-character;
+  font-size: .7em;
+}
+
+.editing-characters p{
+  font-size: 1.7em;
+}
+
+.edit-character-hover, #character-stat-edit, #gear-type {
+  padding: 10px;
```

Comment:
Properties should be ordered border, border-radius, margin-bottom, margin-left, margin-right, margin-top, min-width, padding, text-align, width<br>Each selector in a comma sequence should be on its own single line